tests/test_moment_matching.py

```diff
@@ -0,0 +1,74 @@
+import tensorflow as tf
+import tensorflow_probability as tfp
+tfd = tfp.distributions
+
+from pilco.policies import RBFPolicy
+
+
+class MomentMatchingError(Exception):
+    pass
```

Comment:
This should not be in the test, but in some utils file elsewhere

Reply:
I looked up how [tensorflow does it](https://www.tensorflow.org/api_docs/python/tf/errors), and they have a `tf.errors` for all their errors, so I'll make one for us too.